src/App/Theme.js

```diff
@@ -26,11 +26,21 @@ const theme = createMuiTheme({
     },
   },
   typography: {
+    useNextVariants: true,
     fontFamily: '\'Karla\', sans-serif',
-    title: {
+    h2: {
       fontFamily: '\'Old Standard TT\', serif',
     },
-    display1: {
+    h3: {
```

Comment:
Maybe we should try to figure out how to use comma separated syntax here... Like this:
```
'h1, h2, h3, h4,h5, h6': { ... }
```

Does this not work?

Reply:
No, for some reason it's not working—can we just merge for now?

Reply:
Alright, its just cosmetic